test/Dialect/FIRRTL/lower-types.mlir

```diff
@@ -247,3 +247,35 @@ firrtl.circuit "Foo" {
 
   }
 }
+
+
+// -----
+// https://github.com/llvm/circt/issues/593
+
+module  {
+  firrtl.circuit "top_mod" {
```

Comment:
nit: any reason to not drop this into the omnibus circuit above?

Reply:
so I can put a comment with the issue which is isolated?  Not the strongest reason, but seems somewhat valuable to keep the test cases for each bug separate.

Reply:
I can't read the indentation above... There isn't an ombnibus circuit above. This is fine, but you can drop the outer `module`.